src/update-server.ts

```diff
@@ -1,26 +1,23 @@
 import { logger } from '@runejs/common';
-import { File, Group } from '@runejs/filestore';
 import { ByteBuffer } from '@runejs/common/buffer';
 import { parseServerConfig, SocketServer } from '@runejs/common/net';
-import { FlatFileStore } from '../../filestore';
+import { Archive, File, Group, IndexedFile, FlatFileStore } from '@runejs/filestore/flat-file-store';
+import { StoreConfig } from '@runejs/filestore/config';
 import { FileRequest } from './net/file-request';
 import { UpdateServerConfig } from './config/update-server-config';
 import { UpdateServerConnection } from './net/update-server-connection';
 
 
-export default class UpdateServer {
+export class UpdateServer {
 
     public readonly serverConfig: UpdateServerConfig;
     public fileStore: FlatFileStore;
     public mainIndexFile: Buffer;
 
-    private incomingRequests: string[] = [];
-    private batchLimit: number = 30;
-
     public constructor() {
         this.serverConfig = parseServerConfig<UpdateServerConfig>();
 
-        if(!this.serverConfig.clientVersion) {
+        if(!this.serverConfig.gameVersion) {
             throw new Error(`Update Server supported client version was not provided. ` +
```

Comment:
You've changed `clientVersion` to `gameVersion`, why? Also update the error messages regarding this if there is a need for the change.